src/stories/components/BasicModal/CategoryItem/ArrowAccordionCategory.tsx

```diff
@@ -0,0 +1,99 @@
+import styled from '@emotion/styled';
+import MuiAccordion from '@mui/material/Accordion';
+import MuiAccordionDetails from '@mui/material/AccordionDetails';
+import MuiAccordionSummary from '@mui/material/AccordionSummary';
+
+import { SelectChevronDown } from '@ses/components/svg/select-chevron-down';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import React from 'react';
+
+import type { AccordionProps } from '@mui/material/Accordion';
+import type { AccordionSummaryProps } from '@mui/material/AccordionSummary';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+
+interface Props {
+  style?: React.CSSProperties;
+  category: string;
+  subCategory?: string;
+}
+
+const AccordionCategory: React.FC<Props> = ({ style, category, subCategory }) => {
+  const { isLight } = useThemeContext();
+
+  return (
+    <TransactionHistoryContainer style={style}>
+      <Accordion>
+        <AccordionSummary isLight={isLight}>{category}</AccordionSummary>
+        <AccordionDetails>
+          <div
+            style={{
+              background: '#FBFBFB',
+              boxShadow: '0px 20px 40px rgba(219, 227, 237, 0.4), 0px 1px 3px rgba(190, 190, 190, 0.25)',
+              padding: 64,
+              borderRadius: '0px 0px 6px 6px',
+              textAlign: 'center',
+            }}
+          >
```

Comment:
This should be moved to a styled component in order to avoid inline styles

Reply:
done